modules/dnn/test/test_tf_importer.cpp

```diff
@@ -163,7 +163,7 @@ TEST_P(Test_TensorFlow_layers, pooling)
     runTensorFlowNet("max_pool_even", targetId);
     runTensorFlowNet("max_pool_odd_valid", targetId);
     runTensorFlowNet("ave_pool_same", targetId);
-    runTensorFlowNet("max_pool_odd_same", targetId);
+    runTensorFlowNet("max_pool_odd_same", targetId, false, 3e-5, 3e-4);
```

Comment:
Defaults are: `double l1 = 1e-5, double lInf = 1e-4`

What configuration requires updated thresholds?

Perhaps we should preserve default values for:
- non-OpenCL targets (CPU code)
- for Intel GPU OpenCL